dokka-subprojects/plugin-base/src/test/kotlin/signatures/SignatureTest.kt

```diff
@@ -1366,6 +1366,63 @@ class SignatureTest : BaseAbstractTest() {
         }
     }
 
+    @OnlyDescriptors("#3354")
+    @Test
+    fun `should not render constructor for Any from Wasm sources`() = testRender(
```

Comment:
_nitpick:_ I would prefer to have a more general name for this test, e.g. `should not render constructor without a renderable annotation`

Reply:
There is such test below, though, named a little more explicit.
This specific case is what was happening in original issue. Mostly to check, that it works the same for `kotlin` package and `builtin` class, so Im not sure, what will be the best name here.

Reply:
Among other tests this name made me think about a specific logic for Any (we have such a one) or wasm source set.

Reply:
yeah, I see why there is a confusion... do you have any other idea on how to name this test, so that the intention will be clear?
Does something like this will be better may be `should not render constructor without a renderable annotation for kotlin.Any`? I mean, while there is no special casing for `kotlin.Any`, I would still expect to have a test that checks specifically reported problem here, as kotlin builtins are specific things and so it's good to have tests for them.

Reply:
I think the name below can be reused as the test has the same logic. 
`should not render parameterless constructor with annotation without mustBeDocumented annotation - for kotlin.Any `